### DISCUSSION TITLE
How to sort week days in visualizations?

### DISCUSSION THREAD

**Discussion by pmanandhar** [atoti/atoti#488] [Q&A] [answered]
I would like to know how can one do the custom ordering of the data in the Cube Visualization. I understand that the default ordering is in ascending order. But suppose I want to show Weekdays data in the order of Sunday, Monday, etc. but due to the default ascending ordering, the data is being shown as Friday, Monday and so on.

I used first_members concept:
`l["Week_Day"].comparator == tt.comparator.first_members("Sunday", "Monday","Tuesday","Wednesday","Thursday","Friday","Saturday")`
but it is returning output `False`.
How can one resolve this?

**Comment by bloussou** [comment, marked as answer]
Hello,
You just need to use `=` not `==` :

```diff
- l["Week_Day"].comparator == tt.comparator.first_members("Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday")
+ l["Week_Day"].comparator = tt.comparator.first_members("Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday")
```
